.github/workflows/ci.yml

```diff
@@ -160,7 +160,6 @@ jobs:
     name: Lint and format code
     runs-on: ubuntu-latest
     needs: [check-changed]
-    if: ${{ needs.check-changed.outputs.changed == 'true' }}
```

Comment:
why is this removed

Reply:
I'd like to proceed with the website CI, but it seems this change shouldn't be made, I'll revert it first.